Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM adoptopenjdk/openjdk11
+FROM eclipse-temurin:17-jre-noble
 RUN mkdir /app
-COPY target/xyz-tile-cache-0.4.0.jar /app/xyz-tile-cache.jar
```

Comment:
Looks like this was missed in the last update. Good catch. The way this repo works is every MR merged updates the version so this should now be version 0.6.0 and the top level pom should be updated to reflect that as well